### ISSUE TITLE
Banyule City Council

### ISSUE THREAD

**Issue by mlandauer** [planningalerts-scrapers/issues#515]
This issue has been **automatically** created by PlanningAlerts for [Banyule City Council](https://www.planningalerts.org.au/authorities/banyule)

It uses the scraper [planningalerts-scrapers/banyule](https://morph.io/planningalerts-scrapers/banyule)

Only close this issue once the authority is working again on PlanningAlerts. Otherwise a new issue will just automatically created. Also, if there is a duplicate issue close the other one in favour of this one.

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_4y79h)

**Comment by mlandauer** [comment]
Looks like applications are still available at https://www.banyule.vic.gov.au/Planning-building/Review-local-planning-applications/Planning-applications-on-public-notice

**Comment by mlandauer** [comment]
Oh lordy. Looks like they're using something called [Incapsula](https://en.wikipedia.org/wiki/Incapsula) to block scrapers. Unbelievable.

**Comment by mlandauer** [comment]
@katska this might be really tall order but do you think you could try getting in touch with the council and see if they could unblock the development application page from the "incapsula" system that they're using to block scrapers. I guess it depends on whether they think PlanningAlerts is a valuable enough service to their citizens.

**Comment by katska** [comment]
yup. I'll do that now. Sorry I didn't see before @mlandauer

**Comment by katska** [comment]
I've a draft email to them, and would like to be clear about the request we're making. Would I be asking that they allow requests from morph.io or can we be more specific? @mlandauer do you agree and is that right or would you add something/else?

**Comment by mlandauer** [comment]
@katska Either that they disable incapsula for the application register pages starting at https://www.banyule.vic.gov.au/Planning-building/Review-local-planning-applications/Planning-applications-on-public-notice or that they whitelist morph.io (via IP or whatever), whichever they are willing and able to do.

**Comment by katska** [comment]
draft email as per #557 drafted, awaiting feedback

**Comment by katska** [comment]
I wrote to them last year and didn't hear back. [Sent them a reminder ](https://mail.missiveapp.com/#inbox/conversations/ee1d1071-f9f3-4d04-9e4b-759d866cfef7).

**Comment by katska** [comment]
Council responds: 'It appears that the security architecture provided by our website vendor does not allow the planning application data to be gathered from our website.  This is not an intentional block on Planning Alerts but part of the general website security that we have.  There are no immediate plans to change this.'

**Comment by ianheggie-oaf** [comment]
@jamezpolley - At this point it works on morph with [my proxy](https://morph.io/ianheggie-oaf/banyule), but not the production one. There **was** an error with using early-release platform that stopped it even getting to the proxy, which I have fixed.

I recommend the following to fix the remaining proxy issue, as the block on the old proxies IP has not been released even though we haven't hit the web site for since around 27 Feb 2022:
- https://github.com/planningalerts-scrapers/issues/issues/1071

**Comment by ianheggie-oaf** [comment]
@katska - I recommend the solution above to work around the anti scraping tech given the councils "This is not an intentional block on Planning Alerts" seems to say its not intended to block us, but a side effect of their general protection.

**Comment by jamezpolley** [comment]
I've opened https://github.com/planningalerts-scrapers/issues/issues/1077 to track the remaining proxy issue.
